src/components/Auther__Details.js

```diff
@@ -15,7 +16,8 @@ class Auther__Detail extends React.Component {
     return (
       <div>
         <img  className="Content__Background" src={background} alt="background"/>
-        <img className="Auther__Img" src={john} alt="john smith" ></img>
+        <img className="Auther__Img" src={john} alt="john smith"/>
+        <img class="Content__Sticker1" src={sticker} alt="Background"/>
```

Comment:
move each property to a new line, 
<img classname={}
           src={}
           alt={}/>

Reply:
and apply this to other elements